src/api/rest.tsx

```diff
@@ -455,6 +456,9 @@ export const getApplications = (
       case ApplicationSortBy.TAGS:
         field = "tags.size()";
         break;
+      case ApplicationSortBy.REVIEW:
+        field = "review.deleted,id";
+        break;
```

Comment:
I've tested this change locally and it works returning the expected values; however I'd like to understand:
- What does the `,` (comma) mean for the backend? does it mean: sort by `review.deleted` and r`eview.id`?
- Why are we including the `review.deleted` field? isn't it enough to sort by `review.id`?

Reply:
> I've tested this change locally and it works returning the expected values; however I'd like to understand:
> 
>     * What does the `,` (comma) mean for the backend? does it mean: sort by `review.deleted` and r`eview.id`?
> 
As mentioned in the PR's description, it sorts by "`review.deleted` field and then on `[application.]id` to get consistent ordering within the 2 subsets (`Not started` and `Completed`)"
>     * Why are we including the `review.deleted` field? isn't it enough to sort by `review.id`?
Sorting simply by `review.id` would imply changing the ordering within the 2 subsets (`Not started` and `Completed`) when the direction changes which is not expected.